config/lastmod.yml

```diff
@@ -0,0 +1,298 @@
+---
```

Comment:
not sure if this file should be in source control... a change to any file may lead to a change in this file. Could it be generated during the build pipeline or stored in redis / database?